css/NewEvaluation.css

```diff
@@ -0,0 +1,34 @@
+body {
+    box-sizing: border-box;
+    margin-left: 20%; 
+    margin-right: 20%;
+}
+
+ul {
+    list-style-type: none;
+}
+
+textarea {
+    resize: none;
+}
+
+.evalForms li {
+    float: left;
+    width:16.6%;
+    min-height: 36px;
+}
+
+.clearfix {
+    overflow: auto;
+/*@Alex the clearfix with ::after was the method I used before, it creates an element right after itself on which you can apply the CSS, in this case it was clear:both, in order to clear the float and restore the document normal flow.
```

Comment:
Whoa, talk about inception! Did I ask this once before? Or did you just answer literally moments ago? Or are you some sort of mind reader? :P